examples/rust/components/sqldb-postgres-query/wadm.yaml

```diff
@@ -0,0 +1,42 @@
+---
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: rust-sqldb-postgres-managed
+  annotations:
+    version: v0.1.0
+    description: |
+      Demo WebAssembly component using the wasmCloud SQLDB Postgres provider via the wasmcloud:postgres WIT interface
+spec:
+  components:
+    - name: querier
+      type: component
+      properties:
+        # NOTE: You must run `wash build` for the WebAssembly binary at the relative path below to be present
+        image: file://./build/sqldb_postgres_managed_s.wasm
+      traits:
+        # Govern the spread/scheduling of the actor
+        - type: spreadscaler
+          properties:
+            replicas: 1
+        - type: link
+          properties:
+            target: sqldb-postgres
+            namespace: wasmcloud
+            package: postgres
+            interfaces: [query]
+            target_config:
+              - name: default-postgres
+
+    # Add a capability provider that interacts with the filesystem
+    - name: sqldb-postgres
+      type: capability
+      properties:
+        #image: ghcr.io/wasmcloud/provider-sqldb-postgres:0.1.0
```

Comment:
Let's create an issue to have this updated to be the default once we publish the provider

Reply:
Sure -- I'm going to change this right now, and make a ticket to release the provider

Reply:
https://github.com/wasmCloud/wasmCloud/issues/2198